common/app/model/dotcomrendering/DotcomRenderingDataModel.scala

```diff
@@ -303,10 +303,6 @@ object DotcomRenderingDataModel {
     val config = Config(
       switches = switches,
       abTests = ActiveExperiments.getJsMap(request),
-      commercialBundleUrl =
-        if (ActiveExperiments.isParticipating(StandaloneCommercialBundle)(request))
-          DotcomRenderingUtils.assetURL("javascripts/graun.standalone.commercial.js")
-        else DotcomRenderingUtils.assetURL("javascripts/graun.commercial.dcr.js"),
```

Comment:
Worth noting, @guardian/guardian-frontend-team, that `commercialBundleUrl` would now be in all JSON responses, coming from [`common/app/views/support/JavaScriptPage.scala`].

[`common/app/views/support/JavaScriptPage.scala`]: https://github.com/guardian/frontend/blob/79bab96a825c5dd95d8bbc6fa03a0c0f04782c5c/common/app/views/support/JavaScriptPage.scala#L72